src/worknet/Commands/ContractCommand.Storage.Get.cs

```diff
@@ -0,0 +1,60 @@
+using System.ComponentModel.DataAnnotations;
+using System.IO.Abstractions;
+using McMaster.Extensions.CommandLineUtils;
+using Neo;
+using Neo.BlockchainToolkit.Models;
+using NeoWorkNet.Node;
+
+namespace NeoWorkNet.Commands
+{
+  partial class ContractCommand
+  {
+    // [Command("storage", Description = "Manage neo-worknet values")]
+    // [Subcommand(typeof(Update))]
+    internal partial class Storage
+    {
+      [Command("get", Description = "Update a value in neo-worknet")]
+      internal class Get
+      {
+        readonly IFileSystem fs;
+        public Get(IFileSystem fileSystem)
+        {
+          this.fs = fileSystem;
+        }
+
+        [Argument(0, Description = "Contract name or hash")]
+        [Required]
+        internal string Contract { get; init; } = string.Empty;
+
+        [Argument(1, Description = "Key")]
+        [Required]
+        internal string Key { get; init; } = string.Empty;
```

Comment:
can we add `--json` option to emit the info as JSON?

Reply:
sample output: {"script-hash":"0x50695ace7ab6c5f42d45828d36281bebd83201ee","storages":[{"key":"0xFF","value":"0xABCD"}]} I made this same output format as the list command for consistency. The storage array will always be a length of 1. Let me know if that's ok.

Reply:
I like the consistency